hassio/src/addon-view/config/hassio-addon-config.ts

```diff
@@ -152,6 +152,7 @@ class HassioAddonConfig extends LitElement {
             : html` <ha-yaml-editor
                 @value-changed=${this._configChanged}
                 .yamlSchema=${ADDON_YAML_SCHEMA}
+                .hass=${this.hass}
```

Comment:
can we just pass `hass` when autocomplete is enabled? Otherwise it will just cause extra render cycles

Reply:
We can also remove `@property({ attribute: false })` from `hass` as it doesn't needs to update the element.

Reply:
Applied [your suggestion](https://github.com/home-assistant/frontend/pull/11099#discussion_r798472188), I think we're good 👍🏻